packages/components/app/styles/components/alert.scss

```diff
@@ -0,0 +1,43 @@
+//
+// ALERT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+.hds-alert {
+  background-color: var(--token-color-surface-warning);
+  padding: 1.25rem; // 20px
+}
+
+.hds-alert__header {
+  align-items: center;
+  display: flex;
+}
+
+.hds-alert__icon {
+  margin-right: 0.75rem; // 12px
+}
+
+.hds-alert__title {
+  color: var(--token-color-foreground-warning-on-surface);
+  font-family: var(--token-typography-body-200-font-family);
+  font-weight: var(--token-typography-font-weight-semibold);
+  margin: 0; // Temp override 
+}
+
+.hds-alert__body {
+  color: var(--token-color-foreground-warning-on-surface);
+  font-family: var(--token-typography-body-200-font-family);
+  margin: 0 0 0 2.25rem; // Temp override for 0's, 36px
+}
+
+.hds-alert__footer {
+  align-items: center;
+  display: flex;
+  margin: 1.25rem 0 0 2.25rem; // 20px 0 0 36px
+}
+
+.hds-alert__button {
```

Comment:
[suggestion] it's OK because they're left aligned. in general, it's safer to do something like 
```
.hds-alert__button + .hds-alert__button {
    margin-left: 1.25rem; // 20px
}
```
or even more explicit:

```
.hds-alert__button {
    // generally, you will have more declarations here, so that's why we use the `& + &` pattern below
    & + & {
         margin-left: 1.25rem; // 20px
    }
}
```